apps/condo/domains/user/schema/User.js

```diff
@@ -71,15 +74,27 @@ const User = new GQLListSchema('User', {
             schemaDoc: 'Email. Transformed to lower case',
             type: Text,
             access: access.canAccessToEmailField,
-            kmigratorOptions: { null: true, unique: true },
+            kmigratorOptions: { null: true, unique: false },
```

Comment:
warning! Email should be unique!

Reply:
Email should be unique with user type, as we can have resident and employee with a same email

Reply:
Can complex unique constraint be declared using exposed knex SDL in #524?

Reply:
Added